src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
+     */
+    @Setter
+    @Getter
+    @ToString
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public class ChatRequest {
+        private String model;
+        private List<?> messages;
```

Comment:
:information_source: Using wildcard type '?' in a List is not type-safe and should be avoided. It's better to specify the type explicitly. (Severity: Minor)